README.md

```diff
@@ -1,69 +1,125 @@
-# Theia applications
+<!-- Main Header  -->
+<div align='center'>
+
+<img src="./assets/theia.svg" width="125px">
+
+## Theia Applications - Docker & Electron
+
 [![Build Status](https://travis-ci.org/theia-ide/theia-apps.svg?branch=master)](https://travis-ci.org/theia-ide/theia-apps)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-curved)](https://github.com/theia-ide/theia-apps/labels/help%20wanted)
+[![Questions](https://img.shields.io/badge/Questions-blue.svg?style=flat-curved)](https://github.com/theia-ide/theia-apps/issues?utf8=%E2%9C%93&q=label%3Aquestion+)
+
+Collection of example cloud & desktop applications built using the [Theia](https://github.com/eclipse-theia/theia) platform.
 
-[Theia](https://github.com/theia-ide/theia) is a platform to develop Cloud & Desktop IDEs with modern web technologies.
+</div>
 
-This repository contains example Theia applications based on published Theia extension packages.
+---
 
-- [Theia Docker](#theia-docker)
-  - [How to use `theiaide/theia` image?](#how-to-use-theiaidetheia-image)
-  - [Image Variants](#image-variants)
-    - [`theiaide/theia:latest`](#theiaidetheialatest)
-    - [`theiaide/theia:<version>`](#theiaidetheiaversion)
-    - [`theiaide/theia:next`](#theiaidetheianext)
-    - [`theiaide/theia:<version>-next.<commit hash>`](#theiaidetheiaversion-nextcommit-hash)
-- [Theia Desktop](#theia-desktop)
-- [Contributing](CONTRIBUTING.md)
+<br />
+
+### Outline
+
+- [Docker Image Variants](#docker-image-variants)
+- [Additional Docker Examples](#additional-docker-examples)
+- [Electron Applications](#electron-apps)
+- [Building & Running](#building-&-running)
+- [Tips & Tricks](#tips-&-tricks)
+    - [Build Options](#build-options)
+- [Contributing](#contributing)
 - [License](#license)
 
-## Theia Docker
+<br />
+
+### Docker Image Variants
+
+<img
+    src='./assets/docker.png'
+    alt='docker logo'
+    width="125px"
+/>
+
+| Image Name | Description | Documentation |
+|:---|:---|:---|
+| [theiaide/theia-cpp](https://hub.docker.com/r/theiaide/theia-cpp) | Theia-based C/C++ example application | [docs](./theia-cpp-docker/README.md) |
```

Comment:
What is a reason to order like that?

Reply:
No real reason besides ordering the application names alphabetically.
Should we order differently?

Reply:
Alphabetical probably fine. I want to have `theiaide` as a counterpart of VS Code running on alpine. It would be good to promote it specifically as we did so far.